fonts/fonts.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Does this folder come from PDF functionality?

Reply:
Removed these files.